features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -65,7 +76,55 @@ internal class ImageWireframeHelper(
         return imageWireframe
     }
 
-    private companion object {
-        private const val DRAWABLE_CHILD_NAME = "drawable"
+    @Suppress("NestedBlockDepth")
+    internal fun createCompoundDrawableWireframes(
+        view: TextView,
+        mappingContext: MappingContext,
+        startIndex: Int
+    ): MutableList<MobileSegment.Wireframe> {
+        val result = mutableListOf<MobileSegment.Wireframe>()
+        var currentIndex = startIndex
+        val density = mappingContext.systemInformation.screenDensity
+
+        view.compoundDrawables.forEachIndexed { index, _ ->
```

Comment:
probably we need to add a comment saying that `compoundDrawables` returns an array of drawables in the following order: left, top, right, bottom. It will be easier to understand what is the index.